### DISCUSSION TITLE
All favorites always empty

### DISCUSSION THREAD

**Discussion by Atarinside** [punxaphil/custom-sonos-card#518] [Q&A] [answered]
Hi,

The sonos Card always shows "no favorites found".
![image](https://github.com/user-attachments/assets/cbb22589-f1ad-4c94-9393-27692dc1872b)

But in HA/devellopper tool/status, the favorites are ok :
![image](https://github.com/user-attachments/assets/5a7bedde-804e-46cd-a123-70d9e09ffd8a)

I tried to :
- Remove all favorite/recreate them from the Sonos App (Windows).
- Restart HA
No success so far.

Product : Sonos One SL
Sonos app: Latest : 16.4.2
Sonos OS: S2

Ha version:
Core 2024.10.3
Supervisor 2024.10.2
Operating System 13.2
Interface utilisateur 20241002.3

Everything else is working with the Sonos card:
![image](https://github.com/user-attachments/assets/0c29bbea-b5b9-4bc3-8492-1c224e1e3b78)

Thanks for your help :)

Best regards.

**Comment by punxaphil** [comment]
any errors in the web console?

**Reply by Atarinside** [reply]
No, nothing including the "Sonos" keyword but this (after a HA restart) :

2024-10-23 12:40:43.955 WARNING (ImportExecutor_0) [homeassistant.components.media_player.const] MEDIA_CLASS_DIRECTORY was used from sonos_cloud, this is a deprecated constant which will be removed in HA Core 2025.10. Use MediaClass.DIRECTORY instead, please report it to the author of the 'sonos_cloud' custom integration

**Comment by punxaphil** [comment]
This card doesn't support the sonos_cloud custom integration (never heard about it before). Try uninstalling it and only use the official integration.

**Reply by Atarinside** [reply]
I use it to send TTS messages to my Sonos. It creates a media_player.Sonos_3 but that's all https://github.com/jjlawren/sonos_cloud.

Anyway the official integration is installed and works:
![image](https://github.com/user-attachments/assets/6e8fbf82-636c-4b44-b6d6-9a48b963b57c)

**Comment by punxaphil** [comment]
please validate that the sonos favorites in the built in media player are shown, when you browse media.

**Reply by Atarinside** [reply]
Ok.
I removed the Sonos_cloud integration and restarted HA.

I added the HA build in media player and the Sonos favorites are empty too.
![image](https://github.com/user-attachments/assets/9e985d48-8258-4b96-beb8-f720d8118739)

The music library seems ok.
![image](https://github.com/user-attachments/assets/e6705034-8a69-4db4-abc6-83f44d3a3374)

**Reply by punxaphil** [reply, marked as answer]
Ok, then I am afraid you need to go ask the Sonos integration team

**Reply by Atarinside** [reply]
Ok, thanks for your help.
Best regards

**Comment by Atarinside** [comment]
To follow : https://github.com/home-assistant/frontend/issues/22494
